WordPress/src/main/java/org/wordpress/android/ui/stats/StatsFollowersFragment.java

```diff
@@ -125,69 +125,106 @@ protected void updateUI() {
             ArrayAdapter adapter = new DotComFollowerAdapter(getActivity(), followersModel.getFollowers());
             StatsUIHelper.reloadLinearLayout(getActivity(), adapter, mList, getMaxNumberOfItemsToShowInList());
             showHideNoResultsUI(false);
+
+            if (mTopPagerSelectedButtonIndex == 0) {
+                mTotalsLabel.setText(getTotalFollowersLabel(followersModel.getTotalWPCom()));
+            } else {
+                mTotalsLabel.setText(getTotalFollowersLabel(followersModel.getTotalEmail()));
+            }
+
             if (isSingleView()) {
                 if (followersModel.getPages() > 1) {
-                    mPaginationContainer.setVisibility(View.VISIBLE);
-                    mPaginationText.setText(
-                            String.format(
-                                    getString(R.string.stats_pagination_label),
-                                    followersModel.getPage(),
-                                    followersModel.getPages()
-                            )
+                    mBottomPaginationContainer.setVisibility(View.VISIBLE);
+                    mTopPaginationContainer.setVisibility(View.VISIBLE);
+                    String paginationLabel = String.format(
```

Comment:
Based on the screenshot below, thousands-separators would be good here.

![screen shot 2015-02-12 at 5 52 19 pm](https://cloud.githubusercontent.com/assets/3903757/6178799/8873db40-b2e0-11e4-9ddd-91be325e9620.png)